Compiler/extras/CompilerDevTools/src/CompilerDevTools.jl

```diff
@@ -1,44 +1,76 @@
 module CompilerDevTools
 
 using Compiler
+using Compiler: argextype, widenconst
 using Core.IR
+using Base: isexpr
+
+struct SplitCacheOwner
+    version::Int # use a non-zero value when you don't want to reuse the default cache, e.g. for testing purposes.
```

Comment:
I think this file is just for tests, but note that this approach would be unsound (lead to bad results) if actually used in practice. This should be a mutable singleton struct for correctness in any actual user code